tower-scripts/bin/cicd-control.sh

```diff
@@ -212,6 +212,14 @@ elif [ "${OPERATION}" == "status" ]; then
     /usr/local/bin/autokeys_loader ./aos-cd-cluster-status.sh ${CLUSTERNAME}
   popd
 
+################################################
+# CLUSTER LOG GATHERING
+################################################
+elif [ "${OPERATION}" == "logs" ]; then
```

Comment:
pushd/popd and potentially others output to stdout which could corrupt the tarball generated by gather-logs.sh . Need to move this up the script and exit 0 before more content it output.